mythril/analysis/modules/integer.py

```diff
@@ -215,6 +218,30 @@ def _handle_jumpi(state):
                 )
             )
 
+    @staticmethod
+    def _handle_return(state: GlobalState) -> None:
+        """
+        Adds all the annotations into the state which correspond to the
+        locations in the memory returned by RETURN opcode.
+        :param state: The Global State
+        """
+        stack = state.mstate.stack
+        try:
+            offset, length = get_concrete_int(stack[-1]), get_concrete_int(stack[-2])
+        except TypeError:
+            return
+        for element in state.mstate.memory[offset : offset + length]:
+            if isinstance(element, Expression):
+                for annotation in element.annotations:
+                    if isinstance(annotation, OverUnderflowAnnotation):
```

Comment:
There is a specific getter function, which you can pass the type of annotation you want too receive, could you use that here?

Reply:
Isn't that for state annotations?

Reply:
Right, nvm. We should add that for these annotations too later 😅